src/lib/server/postgres_protocol_handler.cpp

```diff
@@ -0,0 +1,277 @@
+#include "postgres_protocol_handler.hpp"
+
+#include "network_message_types.hpp"
+
+namespace opossum {
+
+static constexpr auto LENGTH_FIELD_SIZE = 4u;
```

Comment:
Each message contains a field indicating the packet's size including itself. Hence, to get the payload's size we need to subtract this field's size. To avoid magic numbers I used `constexpr`. Gonna add a comment in the code.